src/Metadata/Resource/Factory/ExtractorResourceMetadataCollectionFactory.php

```diff
@@ -26,6 +26,9 @@
 use ApiPlatform\Metadata\Put;
 use ApiPlatform\Metadata\Resource\DeprecationMetadataTrait;
 use ApiPlatform\Metadata\Resource\ResourceMetadataCollection;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Symfony\Component\Serializer\NameConverter\CamelCaseToSnakeCaseNameConverter;
```

Comment:
Can we find a way to remove this dependency? Or at least create an interface to abstract it?

Reply:
It should be done in a separate PR as it's used at least 7 times in traits and classes. It's hardcoded (`new CamelCaseToSnakeCaseNameConverter()`), using an interface would require a dependency injection behavior.

Reply:
I don't think there's a point doing an abstraction for this.